larky/src/main/java/com/verygood/security/larky/modules/types/structs/SimpleStruct.java

```diff
@@ -116,4 +104,20 @@ public void debugPrint(Printer p) {
     p.append(")");
   }
 
+  protected Dict.Builder<String, Object> composeAndFillDunderDictBuilder() throws EvalException {
+    StarlarkThread thread = getCurrentThread();
+    StarlarkList<String> keys = Starlark.dir(thread.mutability(), thread.getSemantics(), this);
+    Dict.Builder<String, Object> builder = Dict.builder();
+    for(String k : keys) {
```

Comment:
This isn't a bug that *YOU* did but I think we should have this be sorted to ensure determinism.

Reply:
Done